docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: '2'
```

Comment:
You might as well use version 3.7 while you're here. Nothing in this file isnt compatible with 3.7.

Reply:
@nhumrich done :)